maestro-utils/src/main/kotlin/MaestroTimer.kt

```diff
@@ -23,15 +23,19 @@ object MaestroTimer {
         return null
     }
 
-    fun retryUntilTrue(timeoutMs: Long, delayMs: Long? = null, block: () -> Boolean): Boolean {
+    fun retryUntilTrue(timeoutMs: Long, delayMs: Long? = null, times: Int = 1, block: () -> Boolean): Boolean {
         val endTime = System.currentTimeMillis() + timeoutMs
+        var times = times
         do {
             try {
                 delayMs?.let {
                     sleep(Reason.BUFFER, delayMs)
                 }
                 if (block()) {
-                    return true
+                    times -= 1
+                    if (times == 0) {
```

Comment:
My idea is:

Multiple times calling the same verification method (`block()`) should still return true, as such, times is a variable that could be used to force the verification to return true multiple times before stopping the retry mechanism.

Reply:
That is right but how did we decide 3 would be enough for starting the runner? 🤔

Reply:
Nothing really special, just something about 1

Reply:
I'm actually concerned that we don't know how many iterations it would need really. What about we actually first log if the current condition is not enough for starting the server? Then we would know if the time here needs changed or not? 

Because it can very well happen that the 10 seconds right now are not enough sometimes.

Reply:
What if I increase the timeout to 11 seconds? this would cover 2 more attempts that should be true in the case that only we only get a `true` right at the end of the 10 seconds

Reply:
I am open to increasing it but we actually right now don't even know if there are such cases where the current conditions is not enough. So what about we check that first?

Reply:
That makes sense, but I'll do this later on, is this ok?

Therefore, I've simplified this PR removing the `times` logic and to only look at both the /status call and the appAlive.